.travis_script.sh

```diff
@@ -17,6 +17,7 @@ then
     exit $conf
 fi
 export PYTHONPATH=$(pwd)/tools/pylib/:$PYTHONPATH
+export MAKEFLAGS="-j 2"
 time make || exit
 time make check-unit-tests || exit
```

Comment:
Should we also add `-k`? If for some reason we have multiple errors during make, it might be handy to see as many of them as possible